core/src/main/java/com/amplitude/core/utilities/EventsFileManager.kt

```diff
@@ -74,9 +99,10 @@ class EventsFileManager(
      */
     fun read(): List<String> {
         // we need to filter out .temp file, since it's operating on the writing thread
-        val fileList = directory.listFiles { _, name ->
-            name.contains(storageKey) && !name.endsWith(".tmp")
-        } ?: emptyArray()
+        val fileList =
+            directory.listFiles { _, name ->
+                name.contains(storageKey) && !name.endsWith(".tmp") && !name.endsWith(".properties")
```

Comment:
what's the file end with `.properties`?

Reply:
When it is pure FileStorage, we are using .properties file to store the property. Found this during unit tests so exclude it. In Android this won't happen.